optimum/exporters/onnx/base.py

```diff
@@ -523,6 +528,9 @@ def add_past_key_values(self, inputs_or_outputs: Mapping[str, Mapping[int, str]]
             inputs_or_outputs[f"{name}.{i}.encoder.key"] = {0: "batch_size", 2: encoder_sequence}
             inputs_or_outputs[f"{name}.{i}.encoder.value"] = {0: "batch_size", 2: encoder_sequence}
 
+        if direction == "outputs" and "encoder_last_hidden_state" in inputs_or_outputs:
```

Comment:
Actually, we do not need to enforce an `OrderedDict` all the time. For the inputs, it is already handled, and outputs are `OrderedDict`s.

Reply:
done